samples/highcharts/sonification/chart-earcon/demo.html

```diff
@@ -1,5 +1,12 @@
 <script src="https://code.highcharts.com/highcharts.js"></script>
 <script src="https://code.highcharts.com/modules/sonification.js"></script>
 
-<div id="container" style="max-width:700px; margin: auto"></div>
-<button id="sonify">Click to play chart</button>
+<div id="outer">
+    <div id="overlay">
+      <button class="btn" id="sonify">Click to play chart</button>
```

Comment:
Realizing this a bit late - but we should also have a button to cancel playing. Can we add this below the chart for example once it is running?